src/app/pages/receive/components/receive-items.tsx

```diff
@@ -0,0 +1,18 @@
+import { ButtonProps, Stack } from '@stacks/ui';
+
+import { Caption } from '@app/components//typography';
+
+interface ReceiveItemsProps extends ButtonProps {
+  children: React.ReactNode;
+  title?: string;
+}
+export function ReceiveItems({ children, title }: ReceiveItemsProps) {
```

Comment:
Another name for this could be `ReceiveItemList` which avoids two very similar looking component names (only the `s`)

Reply:
Nice idea, I'll update it thanks!